src/packages/user/user/components/user-document-start-node/user-document-start-node.element.ts

```diff
@@ -0,0 +1,53 @@
+import { css, html, customElement, property, repeat, state, ifDefined } from '@umbraco-cms/backoffice/external/lit';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { UmbDocumentRepository } from '@umbraco-cms/backoffice/document';
+import { DocumentItemResponseModel } from '@umbraco-cms/backoffice/backend-api';
+
+@customElement('umb-user-document-start-node')
+export class UmbUserDocumentStartNodeElement extends UmbLitElement {
+	@property({ type: Array, attribute: false })
+	ids: Array<string> = [];
+
+	@state()
+	_displayValue: Array<DocumentItemResponseModel> = [];
+
+	#itemRepository = new UmbDocumentRepository(this);
+
+	protected async firstUpdated(): Promise<void> {
+		if (this.ids.length === 0) return;
+		const { data } = await this.#itemRepository.requestItems(this.ids);
+		this._displayValue = data || [];
+	}
+
+	render() {
+		if (this.ids.length < 1)
```

Comment:
I do prefere always having { }, it makes it more clear what the intentions are :-)

Reply:
and in this case I would then prefer an else statement for the next part as that would make it easier to read that its one or the other :-)